packages/api/amplify_api/pubspec.yaml

```diff
@@ -2,6 +2,7 @@ name: amplify_api
 description: The Amplify Flutter API category plugin, supporting GraphQL and REST operations.
 version: 0.5.0
 homepage: https://github.com/aws-amplify/amplify-flutter/tree/main/packages/amplify_api
+publish_to: none # until finalized
```

Comment:
We'll be publishing this for dev preview

Reply:
Yes I understand, comment unclear, but I wanted to keep this here for safety at least until API review finished and was thinking we would remove it once API set (for preview stage), then would include in dev preview.